.gitignore

```diff
@@ -37,4 +37,5 @@ dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
 coverage/
-.nyc_output
\ No newline at end of file
+.nyc_output
+STOPKEY
```

Comment:
❓ Was this intentional? Do we really need to ignore `STOPKEY` I thought it was deployed as part of Solr and not living at the root of the repo?

Reply:
maybe it just pops it into whatever your working directory is when you start ddf?

Reply:
I'm guessing most people start ddf from inside of the target directory which is already ignored

Reply:
I removed this change

Reply:
👍